doc/nrf/gs_ins_windows.rst

```diff
@@ -83,20 +83,22 @@ The |NCS| consists of the following repositories:
 Every |NCS| release consists of a combination of these repositories at different revisions.
 
 .. note::
-   The latest state of development is on the master branches of the repositories.
-   Note that this might not be a stable state.
+   The latest state of development is on the master branch of the fw-nrfconnect-nrf repository.
    To ensure a usable state, the fw-nrfconnect-nrf repository defines the compatible states of the other repositories.
-   However, this state is not necessarily tested.
+   However, this state is not necessarily tested. Releases and tags however do guarantee a higher quality level.
 
    Therefore, unless you are familiar with the development process, you should always work with a specific release of the |NCS|.
 
 To manage the combination of repositories and versions, the |NCS| uses :ref:`zephyr:west`.
-The main repository, fw-nrfconnect-nrf, contains a `west manifest file`_ that defines the versions of all repositories.
-This means that fw-nrfconnect-nrf acts as manifest repository (see :ref:`Repository structure <zephyr:west-struct>`), while the other repositories are project repositories.
-The version of the manifest file defines what versions of the project repositories are checked out.
-
-In this way, you can decide to work with a specific |NCS| release either by initializing west with the manifest file for the release or by checking out the manifest file for the release and updating your project repositories to the corresponding state.
-Alternatively, you can work with the latest state of development by using the manifest file from the master branch of the fw-nrfconnect-nrf repository.
+The main repository, fw-nrfconnect-nrf, contains a `west manifest file`_ that defines the versions of all other repositories.
```

Comment:
should we specifically mention the name of the file `west.yml` here ?

Reply:
No, the link points to the file, and this is an overview section.

Reply:
ahh, ok. 
Great then :+1: